crates/apollo-compiler/tests/validation/ignore_builtin_redefinition.rs

```diff
@@ -0,0 +1,256 @@
+use apollo_compiler::schema::SchemaBuilder;
+use apollo_compiler::Schema;
+use expect_test::expect;
+
+#[test]
+fn handles_built_in_scalar_redefinition() {
+    let schema = r#"
+  scalar String
+  scalar Int
+  scalar Float
+  scalar Boolean
+  scalar ID
+
+  type Query {
+    foo: String
+  }
+  "#;
+
+    let errors = Schema::parse_and_validate(schema, "schema.graphql")
+        .expect_err("valid schema")
```

Comment:
i changed the message here (before it said "invalid schema"). `expect_err` panics when an `Err` is expected but we got an `Ok`. So here, instead of getting an invalid schema, we actually get a valid one.

Reply:
I left it as incorrect, and all the other types I added were also slightly incorrect. A bit harder with scalars, so scalars are all the same.